crates/precompile/src/lib.rs

```diff
@@ -159,10 +161,16 @@ impl Precompiles {
     pub fn prague() -> &'static Self {
         static INSTANCE: OnceBox<Precompiles> = OnceBox::new();
         INSTANCE.get_or_init(|| {
-            let precompiles = Self::cancun().clone();
-            // EIP-2537: Precompile for BLS12-381 curve operations
-            // TODO(alexey): add BLS12-381 precompiles
-            // precompiles.extend(bls12_381::precompiles());
+            let precompiles = Self::berlin().clone();
```

Comment:
why was this changed from cancun?

Reply:
it's a mistake, will fix. thanks for noticing